tests/drivers/gpio/gpio_nrf/src/main.c

```diff
@@ -58,4 +58,38 @@ ZTEST(gpio_nrf, test_gpio_high_drive_strength)
 		err);
 }
 
+/*
+ * Nordic Semiconductor specific,
+ * gpio manipulation with disabled NRFX interrupts
+ */
+ZTEST(gpio_nrf, test_gpio_manipulation_nrfx_int_disabled)
+{
+	int response;
+	const struct device *port;
+
+	port = DEVICE_DT_GET(TEST_NODE);
+	zassert_true(device_is_ready(port), "GPIO dev is not ready");
+
+	response = gpio_pin_configure(port, TEST_PIN, GPIO_PUSH_PULL | GPIO_ACTIVE_HIGH);
+	zassert_ok(response, "Pin configuration failed: %d", response);
+
+	response = gpio_pin_set(port, TEST_PIN, 0);
+	zassert_ok(response, "Pin low state set failed: %d", response);
+
+	response = gpio_pin_set(port, TEST_PIN, 0);
+	zassert_ok(response, "Pin hight state set failed: %d", response);
```

Comment:
The value to be set should be 1, not 0. There's also a typo here - "hight".

Reply:
` * @param port Pointer to the device structure for the driver instance.
 * @param pin Pin number.
 * @param value Value assigned to the pin.
 *
 * @retval 0 If successful.
 * @retval -EIO I/O error when accessing an external GPIO chip.
 * @retval -EWOULDBLOCK if operation would block.
 */
static inline int gpio_pin_set(const struct device *port, gpio_pin_t pin,`